<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayList</class>
 <widget class="QWidget" name="PlayList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>845</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>845</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>135</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="SelectAllDir">
      <property name="text">
       <string>选择目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="search">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>搜索本地音乐</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>82</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>本地音乐</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="playAll">
      <property name="text">
       <string>播放全部</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>841</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget::item{width:75px}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
